__tests__/submissions.ts

```diff
@@ -126,7 +127,10 @@ describe('query submissionAvailability', () => {
       {
         url: 'http://abc.com/4',
         userId: '1',
-        createdAt: subDays(new Date().setHours(23), 1),
+        createdAt: zonedTimeToUtc(
+          utcToZonedTime(new Date().setHours(23), 'Europe/Oslo'),
+          'Europe/Oslo',
+        ),
```

Comment:
Its failing currently for me, I think this just moved the needle at which time it will fail :hidethepain:

https://app.circleci.com/pipelines/github/dailydotdev/daily-api/7980/workflows/6e27c68e-fef8-4401-9d48-47300f22f9b3/jobs/31845

Reply:
🥲

Reply:
Yeah, you're right

Reply:
I re-added the old entry, so now there's two. One will work for midnight, and one will work for everything else 😬

Will run test around midnight to verify it works then too. I also tested submissions around midnight, and it seemed to be the correct and not count yesterdays submissions. But I'll verify that again around midnight.